pyconcepticon/api.py

```diff
@@ -115,30 +115,39 @@ def frequencies(self):
                 if concept.concepticon_id:
                     D[concept.concepticon_gloss] += 1
         return D
-
+    
+    def _get_map_for_language(self, language, otherlist=None):
+        self._to_mapping = getattr(self, "_to_mapping", {})
```

Comment:
Could we move initialization of class instances into `__init__`? Code becomes hard to reason about (also for IDEs), when that property isn't met.